content/history/org.inc

```diff
@@ -180,87 +309,91 @@ function displaytrophyyear($trophies, $year){
-            $entry = $entries[$entrytype];
-            $teamlist = $entry->GetTeamList();
-            $time = VOID;
-            foreach($teamlist as $team){
-              $teamtime = $team->time;
-              if($teamtime > 0 && ($teamtime < $time || $time == VOID)){
-                $time = $teamtime;
-              }
-            }
-            $graphArr[$entrytype][$year] = $time;
+      if(isset($entryTimes[$year])) {
+        $times = $entryTimes[$year];
+        foreach ($entrytypes as $entrytype) {
+          if (isset($times[$entrytype])) {
+            $graphArr[$entrytype][$year] = $times[$entrytype];
           } else {
-            $graphArr[$entrytype][$year]= VOID;
+            $graphArr[$entrytype][$year] = VOID;
           }
         }
       } else {
-        foreach($entrytypes as $entrytype){
-          $graphArr[$entrytype][$year]= VOID;
+        foreach($entrytypes as $entrytype) {
+          $graphArr[$entrytype][$year] = VOID;
         }
       }
     }
 
     foreach($entrytypes as $type){
       //remove series for which there are no records in the time range.  the "VOID" points end up being counted as 1.something, so > 100 filters out non-real entries
-      if(array_sum($graphArr[$type])> 100){
+      if(array_sum($graphArr[$type]) > 100){
         $label = "";
         switch($type) {
-          case "1A": $label = "Men's A"; break;
-          case "1B": $label = "Men's B"; break;
-          case "1C": $label = "Men's C"; break;
-          case "1D": $label = "Men's D"; break;
-          case "2A": $label = "Women's A"; break;
-          case "2B": $label = "Women's B"; break;
-          case "2C": $label = "Women's C"; break;
-          case "2D": $label = "Women's D"; break;
+          case "ma": $label = "Men's A"; break;
+          case "mb": $label = "Men's B"; break;
+          case "mc": $label = "Men's C"; break;
+          case "mc": $label = "Men's D"; break;
+          case "wa": $label = "Women's A"; break;
+          case "wb": $label = "Women's B"; break;
+          case "wc": $label = "Women's C"; break;
+          case "wd": $label = "Women's D"; break;
         }
         $chartData->addPoints($graphArr[$type],$label);
         $chartData->setSerieWeight($label,1);
       }
     }
-    $chartTitle = $org->shortname." ".$firstyear."-".$lastyear;
-    $chartFilename = $org->urlkey."Mtrends";
+    $chartTitle = $org["shortname"]." ".$firstyear."-".$lastyear;
+    $chartFilename = $orgUrlKey."Mtrends";
 
     include("content/history/trendchart.inc");
   }
 }
-
 ?>
 
-<h1><?php echo($longname . " " . $shortname); ?></h1>
+<h1><?php
+      echo($org["longname"]);
+      if ($org["shortname"]) {
+        echo (" <span style=\"font-size:.5em\">(".$org["shortname"].")</span>");
+      }
+?></h1>
+
+<p>Founded: <?php echo($org["yearfounded"]); ?></p>
+<?php
+  if($org["orgurl"]) {
+    // We don't store these with a leading "http://" so....
+    // Hopefully everyone redirects to https if they use it.
```

Comment:
It might be worthwhile (not necessarily needed now) for us to just go check these out by hand and update them? There shouldn't be too many, and I'm happy to do the dirty work hahah

Reply:
This is basically a data question for Jeremy.  And maybe a design question of if we print the url or just make "website" a link.

I'm going to resolve it for now but I'm happy to make the change if you and he decide it makes sense to do so.